keps/sig-auth/3299-kms-v2-improvements/README.md

```diff
@@ -396,18 +401,34 @@ FeatureSpec{
 }
 ```
 
+**Beta**
+```go
+FeatureSpec{
+	Default: true,
+	LockToDefault: false,
+	PreRelease: featuregate.Beta,
+}
+```
+
 ###### Does enabling the feature change any default behavior?
 
-No. The v2 API is new in the v1.25 release.
+No. The v2 API is new in the v1.25 release. Furthermore, even with the feature enabled by default, the user needs to explicitly configure a KMSv2 provider to use this.
 
 ###### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement)?
 
 Yes, To disable encryption at rest using the v2 API:
-1. Disable encryption at rest with KMS provider by running through these [steps](https://kubernetes.io/docs/tasks/administer-cluster/kms-provider/#disabling-encryption-at-rest)
-   1. At the end of this step, all the data in etcd will be unencrypted.
-2. Disable the `KMSv2` feature gate.
+1. Add new `identity` provider at the top of encryption config
+1. Restart kube-apiserver
+1. Run storage migration to migrate all the existing encrypted data to use the `identity` provider
+   1. If running `kubectl get <resource> --all-namespaces -o json | kubectl replace -f -` to migrate data, the user can confirm that the migration is complete by observing the kube-apiserver metric `key_id_hash_last_timestamp_seconds` which will no longer contain the keyID hash of the old KEK.
```

Comment:
nit:
```suggestion
   1. If running `kubectl get <resource> --all-namespaces -o json | kubectl replace -f -` to migrate data, the user can confirm that the migration is complete by observing the kube-apiserver metric `key_id_hash_last_timestamp_seconds` which will no longer have updated timestamps for the keyID hash of the old KEK.
```

I believe metrics with the old keyID hash value might still be there but the timestamp of that metric should no longer be updated.

Reply:
> I believe metrics with the old keyID hash value might still be there but the timestamp of that metric should no longer be updated.

I missed mentioning post API server restart here (i'll update it). Post API server restart, there should be no metric for the old keyID in timestamp or counter as it's no longer used.